docs/config/README.md

```diff
@@ -0,0 +1,191 @@
+# Minio Server Configuration Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/) [![codecov](https://codecov.io/gh/minio/minio/branch/master/graph/badge.svg)](https://codecov.io/gh/minio/minio)
+
+In this document we will walk through the Minio Server configuration files. Minio Server stores all the necessary configuration parameters in `${HOME}/.minio/config.json` , following fields provide more detailed understanding of each of these fields and how we can configure them.
+
```

Comment:
How about -- 

In this document we will walk through the Minio Server configuration files. Minio Server stores all the necessary configuration fields in `${HOME}/.minio/config.json`. Following sections provide detailed explanation of each of these fields and how to customize them.